frontend/src/scenes/instance/Licenses/index.tsx

```diff
@@ -78,9 +83,23 @@ export function Licenses(): JSX.Element {
                 caption={
                     <>
                         Here you can add and manage your PostHog enterprise licenses. When you activate a license key,
-                        enterprise functionality will be enabled immediately. Contact{' '}
-                        <a href="mailto:sales@posthog.com">sales@posthog.com</a> to buy a license or if you have any
-                        issues with a license.
+                        enterprise functionality will be enabled immediately.{' '}
+                        <a href="https://posthog.com/pricing" target="_blank">
+                            Click here
+                        </a>{' '}
+                        to grab a license key (first 1 million events free!). Contact{' '}
+                        <a href="mailto:sales@posthog.com">sales@posthog.com</a> if you have any issues with a license.
+                        {billable_events > -1 && !licensesLoading && systemStatus && (
```

Comment:
```suggestion
                        {billable_events > -1 && !licensesLoading && (
```
L75 already considers if systemStatus is not set.